src/generated/types/Table.ts

```diff
@@ -0,0 +1,55 @@
+
+import { Type } from "@sinclair/typebox";
+import { StringUnion } from '../../sunmao-helper';
+
+export const handlerSchema = Type.Array(Type.Object({
+  componentId: Type.String(),
+  type: Type.String(),
+  method: Type.Object({
+    name: Type.String(),
+    parameters: Type.Optional(Type.Object(Type.Any()))
+  })
+}))
+
+export const ColumnSchema = Type.Object({
```

Comment:
May support `fix` property in column?

Reply:
what do you mean? modify row in table?

Reply:
![image](https://user-images.githubusercontent.com/25414733/151121274-c0f0f154-134c-43ca-981f-197f0d17afa3.png)
Sorry, that's typo. It's `fixed` property. I mean that should support columns fixed at left or right.

Reply:
sure